src/KIM/kim_param.cpp

```diff
@@ -57,16 +57,21 @@
 
 #include "kim_param.h"
 
+#include "comm.h"
 #include "error.h"
 #include "fix_store_kim.h"
 #include "force.h"
 #include "input.h"
 #include "modify.h"
 #include "pair_kim.h"
+#include "utils.h"
```

Comment:
like before. list of include statements can be pruned.

Reply:
@akohlmey I updated the include statements